microsoft/testsuites/network/ethtool_suite.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Ethtool
+
+
+@TestSuiteMetadata(
+    area="network",
+    category="functional",
+    description="""
+    This test suite runs the ethtool related network test cases.
+    """,
+)
+class ethtool(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            This test case verifies if ring buffer settings can be changed with ethtool.
+
+            Steps:
+            1. Get the current ring buffer settings.
+            2. Change the rx and tx value to new_values using ethtool.
+            3. Get the settings again and validate the current rx and tx
+                values are equal to the new_values assigned.
+
+        """,
+        priority=1,
+    )
+    def change_ringbuffer_settings(self, node: Node, rx: int, tx: int) -> None:
```

Comment:
The test case cannot accept such customized arguments. Can they be detected from the distro or vm size?

Reply:
It looks this case is to test changes of rx/tx values. Can it get the default value, make some changes, and change it back? It doesn't need any extra arguments.

Reply:
> It looks this case is to test changes of rx/tx values. Can it get the default value, make some changes, and change it back? It doesn't need any extra arguments.

Yes, I was just replying same to the earlier comment. I thought of doing what you said, just that existing LISAv2 test was passing these arguments. But on second thought, I am going to do what you have mentioned.